### DISCUSSION TITLE
Organization/Team tokens

### DISCUSSION THREAD

**Discussion by hauntingEcho** [npm/feedback#94] [Registry] [answered]
Currently, tokens are only assigned at the user level.  In an organization's CD system, this means that either:
* a token is created by a user - it has access to all of that user's organizations, and can have issues if that person later loses access to your organization
* a service account is created using a service email address, which requires additional overhead

Being able to create tokens directly under organizations or teams would be more maintainable.  This is based on a previous forum suggestion here: https://npm.community/t/npm-tokens-organization-tokens/228/4

**Comment by MylesBorins** [comment, marked as answer]
Hey @hauntingEcho . We had a discussion about tokens in last weeks triage meeting and prioritized some work items. I'll bring this up in next weeks meeting when we follow up on the discussion to see if folks from the team want to prioritize this week as well.

There is an existing work item for [Package Scoped tokens](https://github.com/npm/roadmap/issues/10), which while not offering the same level of control as per org I could see it alleviating the concern regarding user scoped tokens being so broad.

**Reply by JounQin** [reply]
Is there any update on https://github.com/npm/roadmap/issues/10? This is very needed for security on CI.

**Reply by MylesBorins** [reply]
@JounQin this is still something that we are exploring but have not been able to immediately prioritize. We'll update the roadmap issue when we have a timeline on releasing package scoped tokens.
